src/content/en/fundamentals/performance/rail.md

```diff
@@ -67,12 +75,34 @@ Understand how users perceive performance delays:
   </tbody>
 </table>
 
-## Response: respond in under 100ms {: #response }
+The perception of performance delays can vary based on mobile hardware
+and network conditions. While loading an experience in 1000ms is plausible on
+a powerful desktop machine over cable, on mobile, ensuring users can load and 
+interact with content in under 5000s is a more [realistic](http://chimera.labs.oreilly.com/books/1230000000545/ch08.html#MOBILE_NETWORK_LATENCY_OVERHEAD) target under slow 3G.
+
+A user on a 2G/3G/4G connection is almost never always on such a connection
+either. Due to packet loss and network variance, their **effective connection
+type** could be significantly slower.
+
+## Goals and guidelines
+
+RAIL offers a set of performance **guidelines** which map closely to set of 
+**goals** regarding user-facing performance metrics. We believe that our
+guidelines can help you achieve the goals.
+
+* **Goals** aim to be timeless
+* Following the **Guidelines** should result in achieving goals, but may
+be specific to current hardware / connection speed distributions.
+
+## Response: respond in under 50ms {: #response }
 
-You have 100ms to respond to user input before they notice a lag.
-This applies to most inputs, such as clicking buttons, toggling form
-controls, or starting animations. This does not apply to touch drags or
-scrolls.
+**Guideline**: You have 100ms to perform work related to transitioning.
+
+**Goal**:
+We want to transition in under this time and suggest you have 50ms to 
```

Comment:
@addyosmani general writing feedback: our style guide recommends avoiding "we" except in certain circumstances: https://developers.google.com/style/person

When an article represents our team's perspective, it's sometimes appropriate to use "we" (but we need to explicitly define who "we" is). In many cases it's more appropriate to use "you", because it clearly demonstrates who needs to take the action